src/agent/onefuzz-task/src/tasks/coverage/generic.rs

```diff
@@ -106,17 +111,32 @@ impl CoverageTask {
         };
 
         let allowlist = self.load_target_allowlist().await?;
+
         let heartbeat = self.config.common.init_heartbeat(None).await?;
-        let mut context = TaskContext::new(&self.config, coverage, allowlist, heartbeat);
+
+        let mut seen_inputs = false;
+
+        let target_exe_path =
+            try_resolve_setup_relative_path(&self.config.common.setup_dir, &self.config.target_exe)
+                .await?;
+        let target_exe = target_exe_path
+            .to_str()
+            .expect("Failed to resolve target_exe relative path");
```

Comment:
This should currently be unreachable, since we deserialized `target_exe` from a JSON file. However, in case things change, let's convert to a `Result` with an accurate error message:
```suggestion
            .ok_or_else(|| anyhow::format_err!("target exe path is not valid unicode"))?;
```